domain-check-2.sh

```diff
@@ -844,15 +847,15 @@ check_domain_status()
         fi
         prints "${DOMAIN}" "Expired" "${DOMAINDATE}" "${DOMAINDIFF}" "${REGISTRAR}"
 
-    elif [ ${DOMAINDIFF} -lt ${WARNDAYS} ] && [ ${DOMAINJULIAN} -gt 0 ]
+    elif [ ${DOMAINDIFF} -lt ${WARNDAYS} ] && [ ${DOMAINJULIAN} -gt 0 ] && [ ${MONTH} -gt 0 ] && [ ${DAY} -gt 0 ] && [ ${YEAR} -gt 0 ]
```

Comment:
String length is more than 80 characters.
IMHO, you need to add a line break.

Reply:
For me it's fine, also I use VS Code with line wrapping. Anyway, fixed as you wanted :smile: